src/seedu/addressbook/data/tag/Tag.java

```diff
@@ -45,6 +45,10 @@ public int hashCode() {
         return tagName.hashCode();
     }
 
+    public String getName() {
```

Comment:
Good that you have added a new getter to read the name of the tag rather than accessing it directly